src/writeData/containers/TelegrafPluginsPage.tsx

```diff
@@ -56,14 +56,15 @@ const TelegrafPluginsPage: FC<RouteComponentProps<{orgID: string}>> = props => {
   )
 
   let thumbnail = (
-    <img data-testid="load-data-details-thumb" src={image || placeholderLogo} />
+    <img data-testid="load-data-details-thumb" src={placeholderLogo} />
   )
-  let pageContent = <></>
-
   if (image) {
-    thumbnail = <img data-testid="load-data-details-thumb" src={image} />
+    thumbnail = (
+      <img data-testid="load-data-details-thumb" src={image} style={style} />
```

Comment:
is `style` ever set? it looks like it's always going to be an empty object

Reply:
Some plugins need the style object to set additional CSS, because their logo asset doesn't work well inside a square box with a dark background. For example, the Alibaba Cloud logo is short and wide and looks extra small unless additional max width is given to it. And the Hashicorp logo needs a white background rather than our default dark background.